docs/next.config.js

```diff
@@ -8,13 +8,19 @@ const { LANGUAGES_SSR } = require('./src/modules/constants');
 
 const workspaceRoot = path.join(__dirname, '../');
 
+/**
+ * @type {'legacy' | 'sync' | 'concurrent'}
+ */
+const reactMode = 'concurrent';
```

Comment:
Apply

```suggestion
const reactMode = 'legacy';
```

before merge.